.github/workflows/lint-php-cs.yml

```diff
@@ -43,7 +43,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Install dependencies
-        run: composer i
+        run: |
+          composer i
+          composer i
```

Comment:
Any reason to run it twice?

Reply:
Yeah. because of using php scoper the composer binary deps are not installed properly the first time :D